sdk/anomalydetector/ai-anomaly-detector/review/ai-anomaly-detector.api.md

```diff
@@ -0,0 +1,112 @@
+## API Report File for "@azure/ai-anomaly-detector"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/identity';
+
+// @public
+export class AnomalyDetectorClient {
+    constructor(endpointUrl: string, credential: TokenCredential | KeyCredential, options?: AnomalyDetectorClientOptions);
+    changePointDetect(body: ChangePointDetectRequest, options?: OperationOptions): Promise<AnomalyDetectorClientChangePointDetectResponse>;
+    readonly endpointUrl: string;
+    entireDetect(body: Request, options?: OperationOptions): Promise<AnomalyDetectorClientEntireDetectResponse>;
```

Comment:
`entireDetect` doesn't feel like the best name to me, hard to guess what `entire` is referring to.

Reply:
Similarly, should `changePointDetect` be `detectChangePoint`?

Reply:
What about `detectEntireSeries`?

Reply:
I like that approximately 100x more, personally.